ssh/tailssh/tailssh.go

```diff
@@ -1517,8 +1522,58 @@ func (ss *sshSession) startNewRecording() (_ *recording, err error) {
 			ss.logf("recording: error starting recording (failing open): %v", err)
 			return nil, nil
 		}
+
+		recorderDisappear := make(chan struct{})
+		go func() {
```

Comment:
can you promote this to its own named method? Both to reduce the size/indentation of the outer method here, and because we try to avoid unnamed long-lived goroutines, as that slows debugging reading stack traces from goroutine dumps.

Reply:
fair shout. done